src/main/java/techItEasy/techItEasy/Exeptions/RecordNotFoundException.java

```diff
@@ -0,0 +1,18 @@
+package techItEasy.techItEasy.Exeptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+public class RecordNotFoundException {
+
+    @ControllerAdvice
```

Comment:
Dit hele stuk van jou mag in je ExceptionsController, terwijl hier mag staan:
public class RecordNotFoundException extends RuntimeException {
    private static final long serialVersionUID = 1L;

    public RecordNotFoundException() {
        super();
    }

    public RecordNotFoundException(String message) {
        super(message);
    }
}